pandas/tseries/offsets.py

```diff
@@ -581,9 +582,44 @@ class BusinessHourMixin(BusinessMixin):
 
     def __init__(self, start='09:00', end='17:00', offset=timedelta(0)):
         # must be validated here to equality check
-        start = liboffsets._validate_business_time(start)
+        if not is_list_like(start):
+            start = [start]
+        if not len(start):
+            raise ValueError('Must include at least 1 start time')
+
+        if not is_list_like(end):
+            end = [end]
+        if not len(end):
+            raise ValueError('Must include at least 1 end time')
+
+        vliboffsets = np.vectorize(liboffsets._validate_business_time)
```

Comment:
this is equivalent to `start = np.array([liboffsets._validate_business_time(x) for x in start])`?  Does the vectorize really make a difference?

Reply:
Yes they are equivalent. Is there a preference to use either way?

Reply:
yes, @jbrockmendel is more idiomatic here.